src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -62,7 +71,12 @@ export default {
 				pageOffset: 0,
 				pageLimit: 6
 			},
-			loans: [],
+			// Default loans for loading animations
+			loans: [
+				{ id: 1 }, { id: 2 }, { id: 3 },
+				{ id: 4 }, { id: 5 }, { id: 6 }
```

Comment:
I think you can just use `0` for all of these id values and that's what we do for the placeholders [in the new homepage loan grid](https://github.com/kiva/ui/blob/main/src/components/Homepage/HomeExp/KivaMultiCategoryGrid.vue#L94).

Reply:
there is an error for the key using `0` for all of these id values
`[Vue warn]: Duplicate keys detected: 'loan-0'. This may cause an update error.`

Reply:
Change line 30 of this file to use index instead of loan ID.

Reply:
I think doing as @dyersituations has suggested will actually solve your issue below. Pretty sure 0 doesn't resolve any loan data but 1-6 "could" resolve loan data...

Reply:
oh yeah, nice suggestion! thanks @dyersituations  and @mcstover